src/Components/CommentsList.jsx

```diff
@@ -0,0 +1,44 @@
+import { useState, useEffect } from "react";
+import { getCommentsByArticleId } from "../Utils/api";
+import { useParams } from "react-router-dom";
+import CommentCard from "./CommentCard";
+
+export default function CommentsList() {
+  const [commentsData, setCommentsData] = useState(null);
+  const [isLoading, setIsLoading] = useState(true);
+  const [error, setError] = useState(false);
+  const { article_id } = useParams();
+
+  useEffect(() => {
+    getCommentsByArticleId(article_id)
+      .then(({ data }) => {
+        setCommentsData(data.data);
+        setIsLoading(false);
+      })
+      .catch(({ response }) => {
+        setError(response);
+        setIsLoading(false);
+      });
```

Comment:
This looks good. Good to see a reusable function and some isLoading state and error pattern.